packages/preview-middleware-client/src/utils/utils.ts

```diff
@@ -0,0 +1,13 @@
+import type {SingleVersionInfo} from '../../types/global';
+import VersionInfo from 'sap/ui/VersionInfo';
+
+/**
+ * Retrieve the UI5 version from sap.ui.core library
+ */
+export async function getUi5Version() {
+    const version = (await VersionInfo.load({library:'sap.ui.core'}) as SingleVersionInfo)?.version;
+    if (!version) {
+        throw new Error('Could not get UI5 version');
```

Comment:
I am not sure we should break the preview when having trouble to read the version. Shouldn't we give a default value back and just log the error in the console? Something like 1.121.0 or so that would lead to the usage of the most common functionality.

Reply:
Fine for me as well. I was just not sure what a good default would be 🐱